src/models/question.model.js

```diff
@@ -119,8 +119,8 @@ class Question {
     return res;
   }
 
-  delete() {
-    // TODO
+  async delete() {
+    return await db.execute(`DELETE FROM question WHERE id = ?`, [this.id]);
```

Comment:
Questions should be deleted, they should be soft deleted (active = false)
```suggestion
    return await db.execute(`UPDATE question SET active = 0 WHERE if = ?`, [this.id]);
    
```

Reply:
It is better this way